apps/sasquatch/src/main/res/values/settings.xml

```diff
@@ -10,8 +10,8 @@
 
     <string name="mobile_center_analytics_state_key">mobile_center_analytics_state_key</string>
     <string name="mobile_center_analytics_state_title">Analytics state</string>
-    <string name="mobile_center_analytics_state_summary_enabled">Analytics is enabled</string>
```

Comment:
This is quite long ;) I'd like it that way since its closer to what we see on Portal and Documentation site.